app/controllers/v0/sessions_controller.rb

```diff
@@ -79,7 +79,7 @@ def multifactor
     # TODO: DEPRECATED
     def identity_proof
       render json: {
-        identity_proof_url: SAML::SettingsService.idme_loa3_url(current_user)
+        identity_proof_url: SAML::SettingsService.idme_loa3_url(current_user, alt_relay: alternate_saml_relay?)
```

Comment:
This might be the right opportunity to remove this deprecated route altogether. It is no longer being used by the front end.